config.py

```diff
@@ -1,7 +1,12 @@
-import logging
+from logging import warning
 import os
```

Comment:
There is no need to change ``import logging``. We generally import the whole logging module.

Reply:
The following does not fit in 80 columns.
    logging.warning('Deprecation warning: environment variable {} is replaced by {}.'
            .format('COBOT_ROOT', 'BOT_ROOT'))

Reply:
You can do ``logging.warning(
'Deprecation warning: environment variable {} is replaced by {}.'
.format('COBOT_ROOT', 'BOT_ROOT'))``

Reply:
And you do not need to change ``import logging`` to ``from logging import warning``

Reply:
got it.